argo-cd-apps/overlays/konflux-public-production/kustomization.yaml

```diff
@@ -15,6 +15,7 @@ namespace: konflux-public-production
 
 patchesStrategicMerge:
   - delete-applications.yaml
+  - delete-dev-only-applications.yaml
```

Comment:
Why not reuse `delete-applications.yaml` for the respective overlays and add the AppSet deletion patch in there? AFAIK it'll have the same effect.

Reply:
I was honestly on the fence about it....  I liked the clarity of the filename indicating what the effect was....  Updated to reuse deleted-applications.